src/public/meeting.ts

```diff
@@ -66,6 +66,17 @@ export namespace meeting {
     tenantId?: string;
   }
 
+  export interface LiveStreamState {
+    /**
+     * indicates whether broadcast is streaming
```

Comment:
this is not specific to broadcast, but for regular meetings. probably better to use meeting here.

Reply:
same for the comment below.

Reply:
updated to "meeting"